File: Yun Visal/Task_Java/Task1/Credit.java
package Task_Java.Task1;
import java.util.Scanner;

public class Credit {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        System.out.print("Enter credit hours: ");
        int cr = s.nextInt();
        System.out.println("Enter the amount of money spent on books: ");
        int b = s.nextInt();
        int pcr = cr*85;
        int tot = pcr+b+65;

        System.out.println("The student enrolled "+cr +" credit hours and 85$ per credit hour = "+ pcr);
        System.out.println("Plus $"+b+" spent on books = " +b);
        System.out.println("Plus a $65 on athletic fee");
        System.out.println("--> Total: $"+tot);
    }
}

File: Yun Visal/Task/Task1/Credit.scala
package Task

object Credit extends App {
  print("Enter credit hours: ")
  val cr = scala.io.StdIn.readDouble()
  println("Enter the amount of money spent on books: ")
  val b = scala.io.StdIn.readDouble()
  val pcr = cr * 85
  val tot = pcr + b + 65

  println("The student enrolled " + cr + " credit hours and 85$ per credit hour = " + pcr)
  println("Plus $" + b + " spent on books = " + b)
  println("Plus a $65 on athletic fee")
  println("--> Total: $" + tot)
}
